puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -0,0 +1,136 @@
+                }
+            }
+        }
+        stage("Setup RVM") {
+            steps {
+
+                configureRVM(ruby)
+
+            }
+        }
+        stage('Configure Environment') {
+            steps {
+
+                dir('foreman') {
+                    addSettings([
+                        organizations: true,
+                        locations: true
+                    ])
+                    addGem()
+                    databaseFile(gemset())
+                }
+
+            }
+        }
+        stage('Configure Database') {
+            steps {
+
+                dir('foreman') {
+
+                    configureDatabase(ruby)
+
+                }
+
+            }
+        }
+        stage('Run Tests') {
+            steps {
+
+                parallel (
+                    'tests': {
+                        dir('foreman') {
+                            withRVM(['bundle exec rake jenkins:katello TESTOPTS="-v"'], ruby)
+                        }
+                    },
+                    'rubocop': {
+                        dir('foreman') {
+                            withRVM(['bundle exec rake katello:rubocop TESTOPTS="-v" --trace'], ruby)
+                        }
+                    },
+                    'react-ui': {
+                        script {
+                            if (fileExists('package.json')) {
+                                sh 'npm install'
+                                sh 'npm test'
+                            }
+                        }
+                    },
+                    'angular-ui': {
+                        dir('foreman') {
+                            withRVM(['bundle show bastion > bastion-version'], ruby)
+
+                            script {
+                                bastion_install = readFile('bastion-version')
+                                bastion_version = bastion_install.split('bastion-')[1]
+                                echo bastion_install
+                                echo bastion_version
+                            }
+                        }
+
+                        sh "cp -rf \$(cat foreman/bastion-version) engines/bastion_katello/bastion-${bastion_version}"
+
+                        dir('engines/bastion_katello') {
+                            sh "npm install npm"
+                            sh "node_modules/.bin/npm install bastion-${bastion_version}"
+                            sh "grunt ci"
+                        }
+                    },
+                    'assets-precompile': { dir('foreman') {
+                            withRVM(['bundle exec rake plugin:assets:precompile[katello] RAILS_ENV=production --trace'], ruby)
+                        }
+                    },
+                )
+
+            }
+        }
+        stage('Test db:seed') {
+            steps {
+
+                dir('foreman') {
+
+                    withRVM(['bundle exec rake db:drop db:create'], ruby)
```

Comment:
should we drop and create in two steps like in https://github.com/theforeman/foreman-infra/pull/438?